src/Sulu/Bundle/AdminBundle/Resources/js/views/Preview/Preview.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {sidebarStore} from '../../containers/Sidebar';
+import Icon from '../../components/Icon/Icon';
+
+type Props = {};
```

Comment:
If possible we are now using exact types for properties:

```javascript
type Props = {||};
```

 This way we also realize if some wrong props are passed.